### PR TITLE
Price broadcasting feature

### PR DESCRIPTION
**Pull request by lmpn**
Create schema for prices for Flux
Removed dependency of EventBus in Flux
Added logic of inserting prices in Flux
Created periodic broadcast in matching engine
Created handler for periodic broadcast in event bus

### CODE REVIEW

File: apps/event_bus/lib/event_bus/price_broadcast.ex [hunk 1]
```diff
@@ -0,0 +1,14 @@
+defmodule EventBus.PriceBroadcast do
+  @moduledoc """
+  A struct representing the payload of :order_queued events.
```
Comment (agramichael) [inline apps/event_bus/lib/event_bus/price_broadcast.ex:3]: Should be ":price_broadcast events"

File: apps/event_bus/lib/event_bus/price_broadcast.ex [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  """
+
+  use TypedStruct
+
+  @typedoc "OrderQueued"
```
Comment (agramichael) [inline apps/event_bus/lib/event_bus/price_broadcast.ex:8]: Same here

File: apps/flux/lib/flux/event_listener.ex [hunk 3]
```diff
@@ -49,6 +50,15 @@ defmodule Flux.EventListener do
     {:noreply, state}
   end
 
+  def handle_info({:cast_event, :price_broadcast, price}, state) do
+    Logger.info("[F] Processing Price: #{inspect(price)}")
+    IO.puts("PBF")
```
Comment (agramichael) [inline apps/flux/lib/flux/event_listener.ex:55]: Already logging the payload so maybe we should drop the `IO.puts`?

Comment (emanuel) [reply apps/flux/lib/flux/event_listener.ex]: Agree let's drop `IO.puts`

**Pull request merged**